triage/Makefile

```diff
@@ -15,12 +15,17 @@
 push:
   bazel run //images/builder -- --project=k8s-testimages --scratch-bucket=gs://k8s-testimages-scratch triage
 
-push_static:
-	gsutil -h 'Cache-Control: no-store, must-revalidate' -m cp -Z -a public-read index.html interactive.js model.js render.js style.css gs://k8s-gubernator/triage/
+STATIC_FILES=index.html interactive.js model.js render.js style.css
 
-push_staging:
-	gsutil -h 'Cache-Control: no-store, must-revalidate' -m cp -Z -a public-read index.html interactive.js model.js render.js style.css gs://k8s-gubernator/triage/staging/
+push-static:
```

Comment:
This is very similar to `push_static` above, might be confusing

Reply:
I think you're mis-reading the diff. `push_static` was renamed

Reply:
Ah yeah you're right